middlewares/content-security-policy/README.md

```diff
@@ -39,7 +39,7 @@ If no directives are supplied, the following policy is set (whitespace added for
     style-src 'self' https: 'unsafe-inline';
     upgrade-insecure-requests
 
-You can use this default with the `useDefaults` option. `useDefaults` is `false` by default, but will be `true` in the next major version of this module.
+You can use this default with the `useDefaults` option. `useDefaults` is `true` by default.
```

Comment:
Could you also make this change in the top-level `README.md`?

Reply:
Amended in https://github.com/helmetjs/helmet/pull/316/commits/102234273c28ecdda6d27c8487214f2d1493411c